doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -4,22 +4,52 @@
 
 .. index:: mclock; configuration
 
-Mclock profiles mask the low level details from users, making it
-easier for them to configure mclock.
+QoS support in Ceph is implemented using a queuing scheduler based on `the
+dmClock algorithm`_. See :ref:`dmclock-qos` section for more details.
+
+.. note:: The *mclock_scheduler* is supported for Bluestore OSDs. For Filestore
+          OSDs the *osd_op_queue* is set to *wpq* and is enforced even if you
+          attempt to change it.
+
+To make the usage of mclock more user-friendly and intuitive, mclock config
+profiles are introduced. The mclock profiles mask the low level details from
+users, making it easier to configure and use mclock.
 
 The following input parameters are required for a mclock profile to configure
 the QoS related parameters:
 
-* total capacity (IOPS) of each OSD (determined automatically)
+* total capacity (IOPS) of each OSD (determined automatically -
+  See `OSD Capacity Determination (Automated)`_)
 
 * an mclock profile type to enable
 
-Using the settings in the specified profile, the OSD determines and applies the
+Using the settings in the specified profile, an OSD determines and applies the
 lower-level mclock and Ceph parameters. The parameters applied by the mclock
-profile make it possible to tune the QoS between client I/O, recovery/backfill
-operations, and other background operations (for example, scrub, snap trim, and
-PG deletion). These background activities are considered best-effort internal
-clients of Ceph.
+profile make it possible to tune the QoS between foreground and background
+operations.
+
+
+.. index:: mclock; mclock clients
+
+mClock Client Types
+===================
+
+Depending on the type of requests handled, mclock clients are classified into
+the buckets as shown in the table below,
```

Comment:
This sentence can be a bit confusing. Can we clarify what we mean by mclock clients here?

Reply:
I have added a couple of sentences to clarify this.